release.json

```diff
@@ -1,7 +1,12 @@
 {
   "nio/DynamicFields": {
     "language": "Python",
-    "version": "0.1.1",
-    "url": "git://github.com/nio-blocks/dynamic_fields.git"
+    "url": "git://github.com//nio-blocks/dynamic_fields.git",
```

Comment:
Oh that's a good point. It should point to the new one but github does forwarding for us so it will still work. Maybe just the new blocks in here should point to the new one.